### PR TITLE
Delete data-proofer-ignore attribute

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation

/kind feature

> /kind hotfix

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by markmandel** [comment]
Ooh same flake one as #3227

```
--- FAIL: TestAllocatorAllocate (2.63s)
    allocator_test.go:122: 
        	Error Trace:	/go/src/agones.dev/agones/pkg/gameserverallocations/allocator_test.go:122
        	Error:      	Expected value not to be nil.
        	Test:       	TestAllocatorAllocate
    allocator_test.go:123: 
        	Error Trace:	/go/src/agones.dev/agones/pkg/gameserverallocations/allocator_test.go:123
        	Error:      	Not equal: 
        	            	expected: *errors.fundamental(Could not find an Allocatable GameServer)
        	            	actual  : <nil>(<nil>)
        	Test:       	TestAllocatorAllocate
    allocator_test.go:124: 
        	Error Trace:	/go/src/agones.dev/agones/pkg/gameserverallocations/allocator_test.go:124
        	Error:      	Should be false
        	Test:       	TestAllocatorAllocate
```

File: docs/governance/templates/release_issue.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ and copy it into a release issue. Fill in relevant values, found inside {}
   - [ ] Create a new file named {version}.md in `/site/content/en/blog/releases`. Copy the draft release content in this file (this will be what you send via email)[refer the previous release file].
   - run `make site-server` frequently to make sure everything looks fine for the release in your localhost
   - [ ] In `site/content/en/docs/Installation/_index.md #agones-and-kubernetes-supported-versions`, for the current version, replace `{{% k8s-version %}}` with hardcoded Kubernetes versions supported by the current version. And add a row for the Agones release version with `{{% k8s-version %}}` as its supported Kubernetes versions.
-  - [ ] Review all `link_test` and `data-proofer-ignore` attributes and remove for link testing [delete `data-proofer-ignore` from the release file of previous version]
+  - [ ] Run `make del-data-proofer-ignore FILENAME={version}-1.md` to remove `data-proofer-ignore` attribute from previous release blog. Review all occurrences of the link_test and data-proofer-ignore attributes globally. Exclude html and release files.
```
Comment (markmandel) [inline docs/governance/templates/release_issue.md:37]: On `FILENAME={version}-1.md` - why the `-1` ?

**Review by markmandel — APPROVED** [review]

File: docs/governance/templates/release_issue.md (hunk 1 above)
Comment (Kalaiselvi84) [reply docs/governance/templates/release_issue.md:37]: In the previous release blog, there will be an occurrence of the 'data-proofer-ignore' attribute that requires removal. If the release version is 1.33.0 then we need to remove this attribute from 1.32.0.md.

Comment (markmandel) [reply docs/governance/templates/release_issue.md:37]: Oooooh! Of course, I was readying that as "append `-1` to the filename" not "subtract a minor version from `{version}`".

Thanks!

**Review by markmandel — APPROVED** [review]

**Comment by markmandel** [comment]
Flake on gke-autopilot-1.25. e2e-stable

```
RBOSE:     gameserverallocation_test.go:736: 
VERBOSE:         	Error Trace:	/go/src/agones.dev/agones/test/e2e/gameserverallocation_test.go:736
VERBOSE:         	Error:      	Not equal: 
VERBOSE:         	            	expected: 100
VERBOSE:         	            	actual  : 95
VERBOSE:         	Test:       	TestGameServerAllocationDuringMultipleAllocationClients
VERBOSE: --- FAIL: TestGameServerAllocationDuringMultipleAllocationClients (66.87s)
VERBOSE: FAIL
```

**Pull request merged**